src/decompressor.rs

```diff
@@ -0,0 +1,116 @@
+use std::ffi::OsStr;
+use std::io::Read;
+use std::path::Path;
+use std::collections::HashMap;
+use std::process::Command;
+use std::process::Stdio;
+use std::process::ChildStdout;
+use globset::{Glob, GlobSet, GlobSetBuilder};
+
+/// A decompression command, contains the command to be spawned as well as any
+/// necessary CLI args
+struct DecompressionCommand {
+    cmd: &'static str,
+    args: Vec<&'static str>
```

Comment:
Since we are building these values in a `lazy_static!`, we don't actually need the `Vec`. We can use an `&'static [&'static str]` instead.